app/models/section.rb

```diff
@@ -20,4 +22,26 @@ def conflicts_with(section)
     end
     false
   end
+  
+  def conflicts
+    Redis.current.smembers id
+  end
+  
+  private
+  def type_cast_periods_start_end
```

Comment:
Removed the type_cast_periods_start_end method, as period numbers should either no longer need to be converted from strings into ints, or this functionality should be done elsewhere (as we discussed).